src/modules/fetcher/__mocks__/FetcherModule.js

```diff
@@ -25,13 +25,18 @@ class FetcherModule {
 
     FetcherModule.currentlyFetching = false
 
-    let errorMessage = 'This result is invalid because it is the first result produced by the FetcherModule.js mock.'
+    const fetchResult = mapValues(
+      targetFilePaths,
+      value => ({ lastUpdate: moment('2016-02-01T10:35:20Z'), url: value, errorMessage: null })
+    )
 
-    return Promise.resolve(reduce(targetFilePaths, (result, value, key) => {
-      result[key] = { lastUpdate: moment('2016-02-01T10:35:20Z'), url: value, errorMessage: errorMessage }
-      errorMessage = null
-      return result
-    }, {}))
+    const paths = Object.keys(fetchResult)
+    const pseudoRandomPath = paths[Math.floor(0.7 * paths.length)]
```

Comment:
the problem here is, that the order of `Object.keys` is arbitrary and therefore your "pseudo" random operation is also arbitrary and can be regarded as actually random.

Reply:
You are right. I sort now by the url. As time and errorMessage is the same for all values this should make it deterministic.